src/app/modules/map/component/map-dialog/map-dialog.component.html

```diff
@@ -20,60 +20,53 @@
+          </app-item-frame-separator>
         </ng-container>
         <ng-container *ngIf="map.bossCount > 0 || (map.bosses && map.bosses.length > 0)">
           <div class="description">
-            <div *ngIf="map.bossCount > 1">
-              {{ 'map.bosses' | translate }} ({{ map.bossCount }})
-            </div>
-            <div *ngIf="map.bossCount <= 1">{{ 'map.boss' | translate }}</div>
+            <div class="light-grey" *ngIf="(map.bossCount || map.bosses.length) <= 1">{{ 'map.map-boss' | translate }}</div>
+            <div class="light-grey" *ngIf="(map.bossCount || map.bosses.length) > 1">{{ 'map.map-bosses' | translate }} ({{ map.bossCount }})</div>
             <ul class="bosses" *ngIf="map.bosses && map.bosses.length > 0">
               <li *ngFor="let boss of map.bosses">{{ boss }}</li>
             </ul>
-            <app-item-frame-separator
-              [item]="data.item"
-              *ngIf="map.encounter || (map.items && map.items.length > 0)"
-            >
+            <app-item-frame-separator [item]="data.item"
+                                      *ngIf="map.encounter || (map.items && map.items.length > 0) || map.url">
             </app-item-frame-separator>
           </div>
         </ng-container>
         <ng-container *ngIf="map.encounter">
-          <div class="description">
-            <div *ngIf="map.encounter">{{ map.encounter }}</div>
-          </div>
-          <app-item-frame-separator [item]="data.item" *ngIf="map.items && map.items.length > 0">
+          <mat-expansion-panel>
+            <mat-expansion-panel-header expandedHeight="20px" collapsedHeight="20px">
+              <mat-panel-title>{{ 'map.map-encounter' | translate }}</mat-panel-title>
+            </mat-expansion-panel-header>
+            <span [innerHtml]="nl2br(map.encounter)"></span>
+          </mat-expansion-panel>
+          <app-item-frame-separator [item]="data.item" *ngIf="(map.items && map.items.length > 0) || map.url">
           </app-item-frame-separator>
         </ng-container>
         <ng-container *ngIf="map.items && map.items.length > 0">
-          <div class="description">
+          <div>
             <span *ngFor="let item of map.items; let last = last">
-              {{ item }}<span *ngIf="!last">,</span>
+              <span [class.light-grey]="(data.item.properties.areaLevel.value.value) > item.dropLevel">{{ item.item }} (Lv {{ item.dropLevel }})</span><span *ngIf="!last">, </span>
             </span>
           </div>
+          <app-item-frame-separator [item]="data.item" *ngIf="map.url">
+          </app-item-frame-separator>
+        </ng-container>
+        <ng-container *ngIf="map.url">
+          <div class="clickable" (click)="onMapClick($event)" matTooltip="{{ (data.settings.evaluateBrowserAlwaysExternal ? 'app.clickable-external-only' : 'app.clickable') | translate }}">
```

Comment:
you did this tooltip different than all the others. you can use [toolTip] instead of using the {{}}

Reply:
I've opted to change the other tooltips rather than this one to be more consistent with how I've used them in other components.